manifests/init.pp

```diff
@@ -36,6 +36,38 @@
   }
   validate_string($tac_plus_template_real)
 
+  if $acl == 'NONE' {
```

Comment:
If we were going from v1.0.0 to v1.1.0, I'd say they should absolutely stay, though keeping them before we hit v1.0.0 seems like dragging beta quality code and more complexity/tests without need.

I understand the position you are in though, perhaps this can be solved on your end through communicating the change and we can thoroughly document it in the README/CHANGELOG?